src/helpers/userSettings.js

```diff
@@ -12,7 +12,11 @@ const passwords = {
   alt11: process.env.ALT11_USER_PASSWORD || 'E-leven',
```

Comment:
please do not change this file. we want this to be in sync with https://github.com/owncloud/web/blob/master/tests/acceptance/helpers/userSettings.js

Reply:
the existing structure does not remember the states, so the vars are moved globally that way. what do you suggest? do we use another separate file for this?

Reply:
it does remembers created users. thats how it's able to delete them at cleanup

Reply:
not requiring the whole helper file was the problem. fixed